C_CPP/Classes & Objects Programming/classes and object video.md

```diff
@@ -0,0 +1,2 @@
+#### CLASSES AND OBJECT VIDEO
+LINK-https://drive.google.com/file/d/1vIXvP5JasjJ-JSZmAKNxWdgIHKKi9iDL/view?usp=sharing
```

Comment:
You can add an image as a thumbnail and add the link to the image. [Here](https://meta.stackexchange.com/questions/38915/creating-an-image-link-in-markdown-format) is how to do that.

Reply:
added, can you please check @Saviour1001